osu.Server.Queues.ScoreStatisticsProcessor/Commands/Queue/ImportHighScoresCommand.cs

```diff
@@ -306,22 +317,25 @@ public async Task<int> OnExecuteAsync(CancellationToken cancellationToken)
                         return -1;
                     }
 
-                    if (elasticQueueProcessor != null)
+                    if (scoreStatisticsQueueProcessor != null)
```

Comment:
As far as I can tell, we should always be pushing to either one queue or the other at this point, right?

In which case, I'd like that to be asserted, and probably use an `if-else` here instead of two `if`s.

Reply:
I pushed https://github.com/ppy/osu-queue-score-statistics/pull/181/commits/0817a9a51d93274ba6823893287531c5ed6251e9 to attempt to address this. Not sure if it's going to be to your liking?